libs/ui/src/icons.tsx

```diff
@@ -94,7 +100,7 @@ function iconColor(theme: UiTheme, color?: IconColor) {
   }[color];
 }
 
-function FaIcon(props: InnerProps): JSX.Element {
+export function FaIcon(props: InnerProps): JSX.Element {
```

Comment:
is this `export` from an older approach? looks like it's not used externally

Reply:
yup, removing, thank you for calling out

Reply:
b1cfb2eb9